addons/l10n_in/data/template/account.tax-in.csv

```diff
@@ -694,3 +694,451 @@
 "","","","","","","","","","","","","","","","","","tax","refund","p10057",""
 "","","","","","","","","","","","","","","","","-100","tax","refund","p11233","l10n_in.tax_tag_cgst_rc"
 "sgst_purchase_5_rc","5% GST RC","GST (RC)","GST 5%","purchase","group","5.0","","","gst_group","","","sgst_purchase_2_5_rc,cgst_purchase_2_5_rc","","0","True","","","","",""
+"cess_non_itc_purchase_5","5% CESS P Non ITC u/s 17(5)","CESS Non ITC u/s 17(5)","CESS 5% Non ITC u/s 17(5)","none","percent","5.0","","","cess_group","","","","","","","","base","invoice","","l10n_in.tax_tag_non_itc_base_cess"

```

Comment:
We have a convention for the names of the taxes were we try to have codes that are as compact as possible, and we have a description field to put the full explanation of the taxes :smile: is it possible to have something more compact ? 

Convention: https://www.odoo.com/odoo/project/967/tasks/3052677?cids=1